internal/documentation/docs/updates/migrate-v5.md

```diff
@@ -0,0 +1,116 @@
+# Migrate to v5
+
+::: tip Alpha Release Available
+**UI5 CLI 5.0 Alpha is now available for testing! 🎉**
+
+Try the alpha release in your projects via: `npm i --save-dev @ui5/cli@next`  
+Or update your global install via: `npm i --global @ui5/cli@next`
+
+**Note:** This is a pre-release version.
+:::
+
+## Breaking Changes
+
+- **All UI5 CLI Modules require Node.js >= 22.20.0 or >= 24.0.0**
+
+- **@ui5/cli: `ui5 init` defaults to Specification Version 5.0**
+
+- **@ui5/project: New Component Type**
+
+	A new project type `component` has been introduced with **Specification Version 5.0**. Component Type projects follow standardized conventions:
+	- Source files in `/src` directory
+	- Test files in `/test` directory
+
+	See [RFC 0018](https://github.com/UI5/cli/blob/rfc-component-type/rfcs/0018-component-type.md) for details.
+
+## Node.js and npm Version Support
+
+This release requires **Node.js version v22.20.0 and higher or v24.0.0 and higher** as well as npm v8 or higher.
```

Comment:
Explicitly mention that 23 is not supported (same as for v21 in the migrate-v4 guide)